files/en-us/web/api/webxr_device_api/fundamentals/index.html

```diff
@@ -19,23 +19,23 @@
+<p>WebXR is an API for web content and apps to use to interface with mixed reality hardware such as VR headsets and glasses with integrated augmented reality features. This includes both managing the process of rendering the views needed to simulate the 3D experience and the ability to sense the movement of the headset (or other motion-sensing gear) and provide the needed data to update the imagery shown to the user.</p>
 
 <p>WebXR additionally provides support for accepting inputs from control devices such as handheld VR controllers or specialized mixed reality gamepads.</p>
 
-<p><em>WebXR is not a rendering technology, and provides no features for actually managing 3D data or rendering it to the display.</em> This is an important fact to keep in mind; while WebXR manages the timing, scheduling, and the various points of view relevant when drawing the scene, it does <em>not</em> actually know how to load and manage models, render and texture them, and so forth. That part is entirely up to you. Fortunately, WebGL and the various WebGL-based frameworks and libraries are available to make it much easier to deal with all of that.</p>
+<p><em>WebXR is not a rendering technology and does not provide features for managing 3D data or rendering it to the display.</em> This is an important fact to keep in mind. While WebXR manages the timing, scheduling, and the various points of view relevant when drawing the scene. It does <em>not</em> know how to load and manage models, nor how to render and texture them, and so forth. That part is entirely up to you. Fortunately, WebGL and the various WebGL-based frameworks and libraries are available to make it much easier to deal with all of that.</p>
 
 <h3 id="How_is_WebXR_different_from_WebVR">How is WebXR different from WebVR?</h3>
 
-<p>WebVR was generally considered to be an experimental API designed to help specification writers determine the best approaches to creating an API for virtual reality on the Web. As browser developers built support for WebVR and allowed developers to experiment, it became clear that in order to finish an API for virtual environments on the web, it would make more sense to start a new specification than to try to "fix" WebVR.</p>
+<p>WebVR was considered an experimental API designed to help specification writers determine the best approaches for creating a virtual reality API on the Web. The advancements of WebVR support in the browsers allowed developers to experiment. But soon it became clear that to finish an API for virtual reality on the web, it would make more sense to start a new specification than to try to "fix" WebVR.</p>
```

Comment:
```suggestion
<p>WebVR was considered an experimental API designed to help specification writers determine the best approaches for creating a virtual reality API on the Web. Browser implementors added WebVR support to browsers, allowing web developers to experiment. But soon it became clear that to finish an API for virtual reality on the web, it would make more sense to start a new specification than to try to "fix" WebVR.</p>
```
Replacing *As browser developers built support for WebVR and allowed developers to experiment, it became clear* with *The advancements of WebVR support in the browsers allowed developers to experiment. But soon it became clear* does not seem to me to be an improvement. Please consider  my suggestion instead.

Reply:
I like your wording more, because it's more specific as to who is adding support to what. Thanks.